NEWS.md

```diff
@@ -1,3 +1,8 @@
+# iSEEfier 1.1.0
```

Comment:
We can push this ahead and already say that the news will refer to the upcoming 1.3.0 - or actually, we can refer to the release and say it is in since 1.2.0

Reply:
@federicomarini do you mean like in this commit [`d2e46b2`](https://github.com/NajlaAbassi/iSEEfier/commit/d2e46b2e655f633e15fd82127a9228f13a973b0e) ?

Reply:
I've edited this one myself, leaving a note in the conversation to check up once you are back from holidays!